modules/test/playwright/tests/commerce/commerce-theme-minium/commerceSearch.spec.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {applicationsMenuPageTest} from '../../../fixtures/applicationsMenuPageTest';
+import {commercePagesTest} from '../../../fixtures/commercePagesTest';
+import {dataApiHelpersTest} from '../../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../../fixtures/loginTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	applicationsMenuPageTest,
+	commercePagesTest,
+	dataApiHelpersTest,
+	loginTest()
+);
+
+test('LPD-29997 Search for products by typing different secification values in global search', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	commerceThemeMiniumPage,
+	page,
+}) => {
+	const site = await apiHelpers.headlessSite.createSite({
+		name: 'Minium',
+		templateKey: 'minium-initializer',
+		templateType: 'site-initializer',
+	});
+
```

Comment:
Use the const {channel, site} = await miniumSetUp(apiHelpers); to create minium. It will take care of deleting the data afterward.